dev/com.ibm.ws.install/resources/com/ibm/ws/install/internal/resources/InstallKernel.nlsprops

```diff
@@ -847,6 +850,26 @@ ERROR_INCOMPATIBLE_FEATURES_SINGLETON=CWWKF1405E: The {0} and {1} singleton feat
 ERROR_INCOMPATIBLE_FEATURES_SINGLETON.explanation=The features cannot be installed at the same time.
 ERROR_INCOMPATIBLE_FEATURES_SINGLETON.useraction=Remove one of the features and try again.
 
+ERROR_FAILED_TO_UNPACK_LICENSE=CWWKF1406E: Unable to unzip the license package with Maven coordinate {0} from the repository at file path {1}.
+ERROR_FAILED_TO_UNPACK_LICENSE.explanation=Unable to unzip license package to the wlp installation.
+ERROR_FAILED_TO_UNPACK_LICENSE.useraction=Ensure you have the correct permissions to read from the repository.
+
+ERROR_FAILED_TO_READ_LICENSE=CWWKF1407E: Unable to process license file at file location {0}.
+ERROR_FAILED_TO_READ_LICENSE.explanation=Unable to process the license file for license acceptance information.
+ERROR_FAILED_TO_READ_LICENSE.useraction=Ensure you have the correct permissions to read from the license file.
+
+ERROR_FAILED_TO_FIND_WEBSPHERE_JSON=CWWKF1408E: Unable to find Websphere JSON file at file location {0}.
```

Comment:
Change message to: `Unable to find the Websphere JSON file at the {0} file location.` or maybe `Unable to find the {0} Websphere JSON file.